assets/compat/config_rules.yaml

```diff
@@ -1,3 +1,5 @@
 ---
-- rule: '!(max-chan-size?) OR !(min-chan-size?) OR (#max-chan-size > #min-chan-size)'
+- rule: "!(max-chan-size?) OR !(min-chan-size?) OR (#max-chan-size > #min-chan-size)"
```

Comment:
What effect does changing from single to double quotes have?

Reply:
No idea, my editor just did it during format-on-save.

Reply:
Are you sure this config rule still works?

Reply:
It must. the change from single to double quotes in this case is a semantically invariant YAML -> YAML transformation. Since we parse YAML before sub-parsing the rules language, it cannot affect the meaning of the rule.